### PR TITLE
Create docker containers with ansible-container instead of docker-compose

### PR DESCRIPTION
**Pull request by mbertrand**
Ansible-container is the equivalent of docker-compose

```
ansible-container build
ansible-container run
```

Ansible will: 
- Generate two docker containers, 1 for mongo and 1 for minerva-girder-gaia
- Generate an admin Girder user
- Generate a default assetstore
- Enable minerva and gaia_minerva plugins
- Enable the gaia analysis widget

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +23,20 @@ Now that Minerva is running, load the page.  At this point you can register an a
...
 
+
+## Option 2: Deploy with ansible-container
+Set the three Girder environment variables in ansible/variables.yml to the correct values for a
+set of working Girder credentials.  This will be the account that Minerva uses to connect with
+Girder.
+    ansible-galaxy install girder.girder
+    export DOCKER_CLIENT_TIMEOUT=600
+    ansible-container build
+    ansible-container run
```
Comment (jbeezley) [inline README.md]: You might want to mention installing ansible-container.  I think `pip install ansible-container[docker]` worked for me.

**Comment by jbeezley** [comment]
I'm getting an error when building:
```
$ DOCKER_CLIENT_TIMEOUT=600 ansible-container build
Building Docker Engine context...	
Starting Docker build of Ansible Container Conductor image (please be patient)...	
Parsing conductor CLI args.
Docker™ daemon integration engine loaded. Build starting.	project=ansible
Building service...	project=ansible service=mongodb
Traceback (most recent call last):
  File "/usr/bin/conductor", line 11, in <module>
    load_entry_point('ansible-container', 'console_scripts', 'conductor')()
  File "/_ansible/container/__init__.py", line 19, in __wrapped__
    return fn(*args, **kwargs)
  File "/_ansible/container/cli.py", line 351, in conductor_commandline
    **params)
  File "/_ansible/container/__init__.py", line 19, in __wrapped__
    return fn(*args, **kwargs)
  File "/_ansible/container/core.py", line 614, in conductorcmd_build
    cur_image_id = engine.get_image_id_by_tag(service['from'])
KeyError: 'from'
Conductor terminated. Cleaning up.	command_rc=1 conductor_id=1bbd8d141c1bbd0d1f4edb01061282757e30e54057664e15d01345e5bcf0634a save_container=False
ERROR	Conductor exited with status 
```

**Comment by mbertrand** [comment]
@jbeezley can you try installing ansible-container==0.3.0 ?  Looks like a merge from development was made for ansible-container in the past week (to 0.9) which broke some things:  https://github.com/ansible/ansible-container/issues/448

**Comment by jbeezley** [comment]
Downgrading to 0.3.0 worked for me.  If you just make a node of that in the README, this LGTM.

**Comment by mbertrand** [comment]
Thanks @jbeezley, I added ansible-container installation instructions in the README

**Review by jbeezley — APPROVED** [review]

**Pull request merged**